html/profile.html

```diff
@@ -10,9 +10,91 @@
 </head>
 
 <body>
-  <h1 class="title-of-page">Setting up Profile.</h1>
 
-  <script src="../script/profile.js"></script>
+  <div class="app">
```

Comment:
class name can be `container` or `profile-container` ?

Reply:
It will be problematic because I have used class profile in that file, so it will be confusing.

Reply:
ok..